.prettierignore

```diff
@@ -1,4 +1 @@
-package.json
-package-lock.json
```

Comment:
Could you please put `package-lock.json` back into the ignore? We don't use these files but in case someone installs with npm instead of yarn I think it would make sense to ignore formatting them.

Reply:
Done 👍